.github/workflows/update_errors.yaml

```diff
@@ -0,0 +1,38 @@
+name: Update Errors specification
+on:
+  repository_dispatch:
+    types: error-spec-change
+
+jobs:
+  snapshotErrorFile:
+    env:
+      CLIENT_ID: ${{ github.event.client_payload.id }}
+      DOWNLOAD_URL: ${{ github.event.client_payload.download_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Fetch OpenAPI doc
+        run: ./scripts/fetch_api.sh ${{ env.DOWNLOAD_URL }}
+      - uses: peter-evans/create-pull-request@v3
+        with:
+          title: "fix(errors): ${{ env.CLIENT_ID }} errors update"
+          token: "${{ secrets.CI_USER_TOKEN }}"
+          commit-message: >
+              fix(errors): ${{ env.CLIENT_ID }} errors update
+          author: "app-services-ci <app-services-ci@users.noreply.github.com>"
+          branch: errors/update-${{ env.CLIENT_ID }}
+          branch-suffix: timestamp
+          reviewers: craicoverflow, wtrocki
```

Comment:
Soon, but not part of this PR. We need to assign the team as reviewers or round-robin, this goes for all SDKs.